requirements.txt

```diff
@@ -1,3 +1,3 @@
-numpy<1.19.0
+numpy
```

Comment:
In applications I always pin NumPy for safety's sake, however in libraries, I usually don't bother, since issues like this one are fairly rare (once every couple years) and version mismatches will cause warnings downstream at install time.

In other words, it's less work to leave the NumPy version number loose.

When a library depends on a new feature in a dependency we can use `>=`, which of course does not cause warnings with later versions.